### PR TITLE
add open62541@1.4.1

### PR DESCRIPTION
**Pull request by udaya2899**

### CODE REVIEW

**Comment by udaya2899** [comment]
Please set the label for the URL check

File: modules/open62541/1.4.1/patches/add_build_file.patch [hunk 1]
```diff
@@ -0,0 +1,17 @@
+diff --git /dev/null BUILD.bazel
+new file mode 100644
+index 00000000..f421d34f
+--- /dev/null
++++ b/BUILD.bazel
+@@ -0,0 +1,10 @@
++cc_library(
++    name = "open62541",
++    srcs = ["lib/libopen62541.a"],
```
Comment (meteorcloudy) [inline modules/open62541/1.4.1/patches/add_build_file.patch:9]: Are you sure `lib/libopen62541.a` is correct? I don't see this file in the source archive.

**Review by meteorcloudy — CHANGES_REQUESTED** [review]

File: modules/open62541/1.4.1/patches/add_build_file.patch (hunk 1 above)
Comment (udaya2899) [reply modules/open62541/1.4.1/patches/add_build_file.patch:9]: let me test this locally again

**Comment by bazel-io** [comment]
Hello @keith, modules you maintain (apple_support, curl, opentelemetry-cpp, opentelemetry-proto, rules_swift) have been updated in this PR. Please review the changes.

**Comment by bazel-io** [comment]
Hello @BalestraPatrick, modules you maintain (apple_support, rules_swift, rules_xcodeproj) have been updated in this PR. Please review the changes.

**Comment by bazel-io** [comment]
Hello @alexeagle, modules you maintain (aspect_rules_js, aspect_rules_lint, rules_oci) have been updated in this PR. Please review the changes.

**Comment by bazel-io** [comment]
Hello @gregmagolan, @jbedard, modules you maintain (aspect_rules_js) have been updated in this PR. Please review the changes.

**Comment by bazel-io** [comment]
Hello @mgred, modules you maintain (bzlparty_tools) have been updated in this PR. Please review the changes.

**Comment by bazel-io** [comment]
Hello @zaucy, modules you maintain (bzlws) have been updated in this PR. Please review the changes.

**Comment by bazel-io** [comment]
Hello @shabanzd, modules you maintain (earcut, gsl) have been updated in this PR. Please review the changes.

**Comment by bazel-io** [comment]
Hello @Vertexwahn, modules you maintain (fmt, glew, xtl) have been updated in this PR. Please review the changes.

**Comment by bazel-io** [comment]
Hello @oxidase, modules you maintain (ofiuco) have been updated in this PR. Please review the changes.

**Comment by bazel-io** [comment]
Hello @jsharpe, modules you maintain (rules_foreign_cc) have been updated in this PR. Please review the changes.

**Comment by bazel-io** [comment]
Hello @luispadron, modules you maintain (rules_ios) have been updated in this PR. Please review the changes.

**Comment by bazel-io** [comment]
Hello @dzbarsky, modules you maintain (rules_itest) have been updated in this PR. Please review the changes.

**Comment by bazel-io** [comment]
Hello @@jin, @@shs96c, @@cheister, modules you maintain (rules_jvm_external) have been updated in this PR. Please review the changes.

**Comment by bazel-io** [comment]
Hello @mark-thm, @theoremlp, modules you maintain (rules_multitool, rules_uv) have been updated in this PR. Please review the changes.

**Comment by bazel-io** [comment]
Hello @thesayyn, modules you maintain (rules_oci) have been updated in this PR. Please review the changes.

**Comment by bazel-io** [comment]
Hello @rickeylev, @f0rmiga, modules you maintain (rules_python, rules_python_gazelle_plugin) have been updated in this PR. Please review the changes.

**Comment by bazel-io** [comment]
Hello @aignas, modules you maintain (rules_python) have been updated in this PR. Please review the changes.

**Comment by bazel-io** [comment]
Hello @Bencodes, modules you maintain (rules_robolectric) have been updated in this PR. Please review the changes.

**Comment by bazel-io** [comment]
Hello @UebelAndre, @illicitonion, @scentini, modules you maintain (rules_rust) have been updated in this PR. Please review the changes.

**Comment by bazel-io** [comment]
Hello @brentleyjones, modules you maintain (rules_swift, rules_xcodeproj) have been updated in this PR. Please review the changes.

**Comment by bazel-io** [comment]
Hello @thii, modules you maintain (rules_swift) have been updated in this PR. Please review the changes.

**Comment by bazel-io** [comment]
Hello @aherrmann, modules you maintain (rules_zig) have been updated in this PR. Please review the changes.

**Comment by bazel-io** [comment]
Hello @mattyclarkson, modules you maintain (toolchain_utils) have been updated in this PR. Please review the changes.

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (grpc, libzip, open62541, pugixml, rules_pkg) have been updated in this PR. Please review the changes.

**Comment by udaya2899** [comment]
I messed up with a faulty rebase. I'm sorry for the notifications.

**Comment by thesayyn** [comment]
Can the @bazel-io be prevented from basically ddos'ing everybody's inboxes?

**Comment by meteorcloudy** [comment]
> Can the @bazel-io be prevented from basically ddos'ing everybody's inboxes?

Yes, fixing in https://github.com/bazelbuild/continuous-integration/pull/2000

**Pull request closed without merging**